src/main/java/com/hillel/items_exchange/service/UserService.java

```diff
@@ -158,4 +162,10 @@ private void addNewPhones(User user, Collection<Phone> phones) {
         phones.forEach(phone -> phone.setUser(user));
         user.getPhones().addAll(phones);
     }
+
+    private List<Child> findDuplicatedChildren(List<Child> fromDB, List<Child> fromRequest) {
```

Comment:
You don't need such method. ```Collections.retainAll()``` makes exactly the same.

Reply:
You totally right =)